scripts/docs/features-overview/build-features-overview.js

```diff
@@ -0,0 +1,491 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const glob = require( 'glob' );
+const chalk = require( 'chalk' );
+
+const DESTINATION_DOCS_PATH = 'docs/builds/guides/integration/features-overview.md';
+const PACKAGE_METADATA_RELATIVE_PATHS = [ 'packages', 'external/*/packages', 'scripts/docs/features-overview/3rd-party-packages' ];
```

Comment:
We need to primarily try to get the 3rd-party packages from node\_modules, as they are installed locally and they will provide their metadata file there in the future.

And then we should fall back to the 3rd-party-packages data that we maintain locally (and hopefully we will be able to remove them soon).